src/components/checkboxes/default.njk

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout-example.njk
+---
+
+{% from 'checkbox/macro.njk' import govukCheckbox %}
+
+{{ govukCheckbox({
+  "idPrefix": "nationality",
+  "name": "nationality",
+  "fieldset": {
+    "legendHtml": '<h1 class="govuk-heading-xl">What is your nationality?</h1>',
+    "legendHintText": "If you have dual nationality, select all options that are relevant to you."
+  },
+  "items": [
+    {
+      "value": "british",
+      "text": "British"
+    },
+    {
+      "value": "irish",
+      "text": "Irish"
+    },
+    {
+      "value": "other",
+      "text": "Citizen of another country"
+    }
+  ]
+}) }}
```

Comment:
A more of a general question but should we have more variants of the checkboxes here? Such as having a disabled and "None" option etc. Or maybe that's coming later.

Reply:
Yes I think this will be a content decision - We'll do a bigger content and examples review once everything is in.